api/src/main/java/com/codeforcommunity/dto/team/InviteUserRequest.java

```diff
@@ -0,0 +1,58 @@
+package com.codeforcommunity.dto.team;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.HandledException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class InviteUserRequest extends ApiDto {
+
+  private Integer teamId;
+  private Map<String, String> users;
```

Comment:
Do we really want to validate the fields of each of the InviteContacts? This would result in an error being thrown if any of the users have an invalid email or leave a value none. The API Spec says in this case an email would just not be sent, and nothing will be changed in the database.